bundles/binding/org.openhab.binding.diyonxbee/src/main/java/org/openhab/binding/diyonxbee/internal/FormatUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.openhab.binding.diyonxbee.internal;
```

Comment:
This class is missing its copyright header as well as @since and @author doc comments.